### PR TITLE
Add frontend deployment config 

### PR DESCRIPTION
**Pull request by kgaikwad**
fixes [RHIROS-27](https://issues.redhat.com/browse/RHIROS-27)

### CODE REVIEW

File: .travis/custom_release.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+set -x
+
+#!/bin/bash
+set -e
+set -x
```
Comment (apuntamb) [inline .travis/custom_release.sh:7]: do we need this twice or just a typo?

File: .travis/custom_release.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+if [ "${TRAVIS_BRANCH}" = "master" ]
+then
+    for env in ci
+    do
+        echo "PUSHING ${env}-beta & ${env}-stable"
+        rm -rf ./dist/.git
+        .travis/release.sh "${env}-beta"
+        .travis/release.sh "${env}-stable"
```
Comment (patilsuraj767) [inline .travis/custom_release.sh:16]: Not sure where this `.travis/release.sh` script is but seems like all the other services are doing the same.

**Review by patilsuraj767 — CHANGES_REQUESTED** [review]
As per the documentation, Prerequisite is to add the ROS app [here](https://github.com/RedHatInsights/cloud-services-config/blob/ci-beta/main.yml) but I cannot see the PR raised in `cloud-services-config` repository.

File: .travis/custom_release.sh (hunk 1 above)
Comment (kgaikwad) [reply .travis/custom_release.sh:7]: Typo! we don't need it twice. removing it

File: .travis/custom_release.sh (hunk 2 above)
Comment (kgaikwad) [reply .travis/custom_release.sh:16]: Script resides here [https://github.com/RedHatInsights/insights-frontend-builder-common](https://github.com/RedHatInsights/insights-frontend-builder-common)

**Comment by kgaikwad** [comment]
@patilsuraj767, 
Created pull-request on `cloud-services-config` side:
Link - [https://github.com/RedHatInsights/cloud-services-config/pull/346](https://github.com/RedHatInsights/cloud-services-config/pull/346)

**Review by patilsuraj767 — APPROVED** [review]

**Comment by apuntamb** [comment]
Merging as this is currently, not dependent on [RedHatInsights/cloud-services-config#346](https://github.com/RedHatInsights/cloud-services-config/pull/346).
Thanks @kgaikwad @patilsuraj767

**Review by apuntamb — APPROVED** [review]

**Pull request merged**